test/controllers/api/v1/commission_histories_controller_test.rb

```diff
@@ -44,4 +44,17 @@ def setup
     json = response_to_json(@response.body)
     assert_equal 2, json.size
   end
+
+  test 'When reaching the commission changes api \
+        with correct token and csv format\
+        should return 200' do
+
+    path = api_v1_commission_histories_index_path(network: "testnet") + ".csv"
+    get path, headers: { "Token" => @user.api_token }
+
+    assert_response :success
+    assert_equal "text/csv", response.content_type
+    csv = CSV.parse response.body # Let raise if invalid CSV
+    assert csv
```

Comment:
In my opinion, it would be good to check if at least one header in the csv file is present or check a number of rows that the file contains.